app/templates/cookies.html

```diff
@@ -123,173 +123,178 @@ <h3 class="ons-u-mt-l">{{ _('Strictly necessary cookies') }}</h3>
                     ]
                 }) }}
             <p>{{ _('They always need to be on.') }}</p>
-            {% call onsCollapsible({
-                "id": "cookies-essential-details",
-                "classes": "ons-u-mb-l",
-                "title": _("Learn more about these essential cookies"),
-                "button": {
-                    "close": _("Hide this"),
-                    "contextSuffix": "content"
-                }
-            }) %}
-                <h4>{{ _('Your progress') }}</h4>
-                <p>{{ _('We’ll set a cookie to remember your progress while logged in. These cookies do not store your
-                    personal data.') }}</p>
-                {{ onsTable({
-                        "ths": [
+        </form>
+    </div>
```

Comment:
Instead of closing them here, `<form>` & `<div>` would need to be closed at the end, right before the `endblock`. This should resolve the issue with the "Return to previous page" backlink so it's all in one form instead.

Reply:
Thanks!